packages/network/package.json

```diff
@@ -34,66 +34,40 @@
     "test": "tsc --noEmit && jest --forceExit"
   },
   "dependencies": {
-    "@latticexyz/common": "workspace:*",
-    "debug": "^4.3.4",
-    "lodash": "^4.17.21",
-    "viem": "^0.3.18"
-  },
-  "devDependencies": {
     "@ethersproject/abi": "^5.7.0",
     "@ethersproject/providers": "^5.7.2",
     "@improbable-eng/grpc-web": "^0.15.0",
+    "@latticexyz/common": "workspace:*",
     "@latticexyz/recs": "workspace:*",
     "@latticexyz/schema-type": "workspace:*",
     "@latticexyz/services": "workspace:*",
     "@latticexyz/solecs": "workspace:*",
     "@latticexyz/store": "workspace:*",
     "@latticexyz/utils": "workspace:*",
-    "@protobuf-ts/grpcweb-transport": "^2.7.0",
+    "async-mutex": "^0.3.2",
+    "debug": "^4.3.4",
+    "ethers": "^5.7.2",
+    "lodash": "^4.17.21",
+    "mobx": "^6.7.0",
+    "nice-grpc-web": "^2.0.1",
+    "rxjs": "^7.5.5",
+    "threads": "^1.7.0",
+    "viem": "^0.3.18"
+  },
+  "devDependencies": {
     "@types/debug": "^4.1.7",
     "@types/google-protobuf": "^3.15.6",
     "@types/jest": "^27.4.1",
     "@types/lodash": "^4.14.182",
     "@types/node": "^18.15.11",
-    "async-mutex": "^0.3.2",
-    "ethers": "^5.7.2",
     "fake-indexeddb": "^4.0.0",
-    "google-protobuf": "^3.21.0",
```

Comment:
I couldn't find any usages of this and the example app seems to run locally without issue.

Reply:
`google-protobuf` is a peer dependency of `@improbable-eng/grpc-web`, which is a dependency of `nice-grpc-web`. It doesn't look like `nice-grpc-web` is installing `google-protobuf`, but it also seems like it works without it (and pnpm doesn't complain about missing indirect peer dependencies), so down to remove it for now.